build.gradle

```diff
@@ -170,7 +170,7 @@ dependencies {
     implementation 'com.google.guava:guava:32.1.1-jre'
     implementation 'org.greenrobot:eventbus:3.2.0'
     implementation 'commons-cli:commons-cli:1.3.1'
-    implementation 'org.bouncycastle:bcprov-jdk15to18:1.75'
+    implementation 'org.bouncycastle:bcprov-jdk15to18:1.78'
```

Comment:
```suggestion
    implementation 'org.bouncycastle:bcprov-jdk15to18:1.78.1'
```

AFAIK OpenSearch Core for 1.3 uses 1.78.1

Reply:
Updated this and the other 2 instances in the integration test dependencies. Thanks @reta!